test/helpers/fake_hubspot_api.js

```diff
@@ -0,0 +1,105 @@
+const nockHelper = require('./nock_helper')
+const applicationId = process.env.APPLICATION_ID || 12345
+const appId = process.env.APP_ID || 23456
+const userId = process.env.USER_ID || 23456
```

Comment:
This is minor but I think it would be helpful to use a different value for the `appId` and the `userId` so there isn't a false sense of similarity between these two values.